include/aspect/particle/particle.h

```diff
@@ -141,11 +141,29 @@ namespace aspect
         Particle (const void *&begin_data,
                   const unsigned int data_size);
 
+#ifdef DEAL_II_WITH_CXX11
         /**
-         * Destructor for Particle
+         * Copy constructor for Particle, creates a particle from an existing
+         * one.
          */
-        virtual
-        ~Particle ();
+        Particle (const Particle<dim> &particle);
```

Comment:
Am I correct that you need to declare the regular copy constructor here because that's required if you *also* have a move constructor, but that you don't need it if not in C++11 because then you can just let the compiler generate the default copy constructor?